### DISCUSSION TITLE
Filtering k8s health check spans from getting generated

### DISCUSSION THREAD

**Discussion by karthikinamanamelluri** [open-telemetry/opentelemetry-java-instrumentation#3121] [Q&A]
Hello,

Is there a way to not capture the traces  generated due to health check api calls when using the agent (at the instrumentation level itself)?
Health checks are generating a lot of spans and i would like to not get them. 

We could say to simply ignore them, but one other challenge is that when parent sampling is enabled, the health check traces would add up to some of the ratio enabled, which otherwise could capture an actual request.

Can someone please let me know if there is any solution to overcome this problem?

**Comment by trask** [comment]
hey @karthikinamanamelluri! this is not implemented yet, check out (and subscribe to) #1060 for updates on this feature
